src/selectedtests/task_mappings/create_task_mappings.py

```diff
@@ -419,7 +420,14 @@ def _create_task_map(tasks: List[Task]) -> Dict:
     :param tasks: List of tasks to map.
     :return: Dictionary of tasks by display_name.
     """
-    return {task.display_name: task for task in tasks}
+    execution_tasks_map = {
```

Comment:
Does this need to be a map? Would it make more sense for it to be a set?

Reply:
For the larger builds (containing large numbers of large display tasks) a map is 14 times faster for more or less the same syntax.

Reply:
A map is faster than a set for this? That seems really surprising.

Reply:
I went back and looked at my testing and it looks like I tested lists twice rather than list, sets and dicts.

A set is more appropriate / clear for this (and the same speed) so I going to change to that.